### DISCUSSION TITLE
Sending commands to NUT server

### DISCUSSION THREAD

**Discussion by MatejKovacic** [hassio-addons/app-nut#248] [Q&A] [answered]
Are there any plans for adding the possibility to send commands to NUT server?
It would be useful to send commands to the UPS for starting a quick test or set the beeper on or off.

More is described here: https://community.home-assistant.io/t/nut-commands-upscmd/401107/2

**Comment by frenck** [comment, marked as answer]
As per the feature request linked:

> It would be great if the nut integration could send commands.

This repository Is **not** the integration, but the add-on.
I think you have mixed up things.

../Frenck

**Reply by Santanachia** [reply]
Does this add-on allow the command to be received from the integration and sent to the device?

**Comment by MatejKovacic** [comment]
Ah. Could you direct me to the NUT integration issues? I find this Github repo a little confusing...
